src/cbapi/response/models.py

```diff
@@ -294,6 +294,21 @@ def refresh(self):
             self._info = result[0]
         self._last_refresh_time = time.time()
         return True
+    
+    def set_ignored(self, ignored_flag=True):
+        payload = {"updates": {"is_ignored": ignored_flag, "requested_status": "False Positive"}}
```

Comment:
If we do a set_ignored(False) here, shouldn't the "requested_status" be set to something other than "False Positive"? Just saying.

Reply:
The possible values for `"requested_status"` are [Resolved, Unresolved, In Progress, or False Positive.](https://developer.carbonblack.com/reference/enterprise-response/6.3/rest-api/#bulk-update-alerts)

When doing `set_ignored(ignored_flag=False)`, I would suggest `"requested_status"` to be Unresolved

Reply:
Does the ignored flag even do anything? I believe it was added for a future enhancement that hasn't ever been introduced (method was [introduced](https://cbapi.readthedocs.io/en/latest/changelog.html) in cbapi v1.1.1). That said, if an alert is ignored, wouldn't it be ignored from an existing "Unresolved" status?